cmd/humioctl/repos_add_user.go

```diff
@@ -0,0 +1,35 @@
+package main
+
+import (
+	"os"
+
+	"github.com/spf13/cobra"
+)
+
+func newReposAddUserCmd() *cobra.Command {
+	cmd := cobra.Command{
+		Use:   "add-user [flags] <repo> <username> [group - default: Member]",
+		Short: "Adds a user to a repository",
+		Args:  cobra.RangeArgs(2, 3),
+		Run: func(cmd *cobra.Command, args []string) {
+			repoName := args[0]
+			userName := args[1]
+			group := "Member"
+
+			if len(args) == 3 {
+				group = args[2]
+			}
+
+			if group != "Member" && group != "Admin" && group != "Eliminator" {
+				cmd.Println("group was invalid must be one of: Member, Admin or Eliminator")
+				os.Exit(1)
+			}
```

Comment:
It's a bit annoying that Humio is case-sensitive in this regard. I would really want to accept "admin" as input and have it converted to "Admin".

Might consider adding a `(*DefaultGroupEnum).ParseString(string) bool` method that does parsing, which can also be used by users of the `api` package.